### DISCUSSION TITLE
Trying to register PnP in GCCH

### DISCUSSION THREAD

**Discussion by pjacksone** [pnp/powershell#3701] [General]
New to this module, so I am unsure where to start. Trying to pull a SharePoint template and push it to another site, but I think the register-pnp is stopping my ability to push the folders and pages to the site.

Is there process for installing and going through the process for this PnP Powershell module?

**Comment by jakeatsb** [comment]
Try this - https://pnp.github.io/powershell/articles/authentication.html#register-your-own-azure-ad-app.  I'm running into an issue where it authenticates but then never prompts for the perms but it's at least authenticating.  Still troubleshooting the perm window not generating to create the app but auth seems to work.
